vip-cache-manager/vip-cache-manager.php

```diff
@@ -361,33 +443,72 @@ function queue_purge_urls_for_term( $term ) {
 		 * example.com/category/news/page/2
 		 * example.com/category/news/page/3
 		 *
+		 * @access private Please do not hook this filter at the moment
 		 * @param int The maximum page to purge from each term archive
 		 * }
 		 */
-		$max_pages = apply_filters( 'wpcom_vip_cache_purge_urls_max_pages', 2 );
+		$max_pages = apply_filters( 'wpcom_vip_cache_purge_urls_max_pages', 2, $term );
 
 		// Set some limits on max and min values for pages
-		$max_pages = max( 1, min( 5, $max_pages ) );
+		$max_pages = max( 1, min( 20, $max_pages ) );
```

Comment:
Addressed in https://github.com/Automattic/vip-go-mu-plugins/pull/247/commits/ba29b31ce8b6ee0cdbfcde4a7181f5e2c7f53e39